config.go

```diff
@@ -156,6 +156,7 @@ type config struct {
 	// IPC options
 	PipeTx            *uint `long:"pipetx" description:"File descriptor or handle of write end pipe to enable child -> parent process communication"`
 	PipeRx            *uint `long:"piperx" description:"File descriptor or handle of read end pipe to enable parent -> child process communication"`
+	NamedPipeRx       string `long:"namedpiperx" description:"Named pipe (in windows) or unix socket of read end pipe to enable wallet -> client process communication"`
```

Comment:
sockets, fifos, etc. are bidirectional, so there's no need for a tx/rx distinction.  The winio package emulates this using two named pipes.  Should simply name this `Pipe` and let the implementation be platform specific (named pipes are not usually called that on unix).